package com;

public class TryWithMultipleCatchBlock {

	public static void main(String[] args) {
		try {
			int res = 10/0;
		} catch (ArithmeticException e) {
			System.out.println("Divided by Zero "+e);
		}catch (ArrayIndexOutOfBoundsException e) {
			System.out.println("Array index "+e);
		}
	}

}
